cloudant-client/src/test/java/com/cloudant/tests/CloudantClientHelper.java

```diff
@@ -32,11 +33,11 @@
     //some tests need access to the URI with user info (e.g. replication)
     public static final String SERVER_URI_WITH_USER_INFO;
     //some tests need access to the credentials (e.g. auth interceptors, vcap)
-    public static final String COUCH_USERNAME;
-    public static final String COUCH_PASSWORD;
-    public static final String COUCH_HOST;
+    static final String COUCH_USERNAME;
```

Comment:
Should we rename these to SERVER_URL, SERVER_USER, and SERVER_PASSWORD?

Reply:
Well these names are independent of the environment variables that follow the `SERVER_` convention (and system properties), but that's fine I can rename to bring things more inline.
See bc26da61330e802a8f09094877e3f902032fddc8.